packages/doc-spa/src/pages/index.js

```diff
@@ -1,51 +1,51 @@
-import React from 'react';
-import clsx from 'clsx';
-import Layout from '@theme/Layout';
-import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import useBaseUrl from '@docusaurus/useBaseUrl';
-import styles from './styles.module.css';
+import React from "react";
```

Comment:
Didn't we have single quote as default? @akhilmhdh

Reply:
This is why we need a centralized eslint config. To make the style uniform and not depend on editor. I actually tried everything from vscode setting to editorconfig. Else need to modify editor config